### ISSUE TITLE
Update Travis CI to 3.9 final once released

### ISSUE THREAD

**Issue by johnthagen** [JoelLefkowitz/drf-yasg#15]
Follow on to #14 / #52

Travis CI has not yet released a Python 3.9 final image: https://travis-ci.community/t/python-3-9-0-build/10091

Once this happens, we should update the Travis config from `3.9-dev` to `3.9`.

**Comment by JoelLefkowitz** [comment]
I think this was done to completion in #11. Please reopen this issue if I'm wrong.

**Comment by johnthagen** [comment]
@JoelLefkowitz This should be reopened. We are still at 3.9-dev on Travis as Travis has not release 3.9 final image yet:

https://github.com/JoelLefkowitz/drf-yasg/blob/9ede71d54a8e266f8a5c16739cb8987a62998114/.travis.yml#L34

**Comment by JoelLefkowitz** [comment]
Ah I understand, hopefully they'll release a stable 3.9 soon

**Comment by johnthagen** [comment]
Yes, I would expect them to have it ready in a week or two at the latest.

**Comment by johnthagen** [comment]
https://github.com/axnsan12/drf-yasg/pull/665
